.gitignore

```diff
@@ -0,0 +1,20 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
```

Comment:
Add .DS_Store to your .gitignore or if you find you are constantly forgetting to do that add it to a global gitignore.